### PR TITLE
Add how to debug section

### LINKED ISSUE #125: Add a "how to debug dask-jobqueue" section in the docs
Based on this comment https://github.com/dask/dask-jobqueue/issues/123#issuecomment-412412517, or that of @jhamman in #99.

Some hints to see what is going wrong:
- print(cluster.jobs_script(). Launch it manually.
- Activate debug mode
- use job queuing system commands to look for any running jobs
- use job queuing system commands to inspect stderr or stdout of jobs
...

### PR DESCRIPTION
**Pull request by guillaumeeb**
Fixes #125 

cc @jhamman.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -216,3 +216,72 @@ In dask-distributed, a ``Worker`` is a Python object and node in a dask
...
+Dask jobqueue has been developed and tested by several contributors, each of
+us having a given HPC system setup to work on: a job scheduler in a given
+version running on a given OS. Thus, in some specific case, it might not work
+out of the box on your system. This section provides some hints to help you
+sort what may be going wrong.
+
+Checking job script
+~~~~~~~~~~~~~~~~~~~
+
+Often, the first thing to do is checking if the job script generated to submit
```
Comment (jhamman) [inline docs/index.rst]: Maybe start this section with something more like: 

Dask-jobqueue submits "job scripts" to your queueing system. Inspecting these scripts often reveals errors in the configuration of your Cluster.

File: docs/index.rst [hunk 2]
```diff
@@ -216,3 +216,72 @@ In dask-distributed, a ``Worker`` is a Python object and node in a dask
...
+jobs launching dask workers looks like your usual submission script, in
+particular the header containing ``#PBS``, ``#SBATCH`` or other directives.
+This can be done easily once you've created a cluster object:
+
+.. code-block:: python
+
+   print(cluster.job_script())
+
+Next move if you don't see any problem is to actually try to run this script,
```
Comment (jhamman) [inline docs/index.rst]: If everything in job script appears correct, the next step is to try to submit a test job using the script. You can simply copy and paste...

**Comment by guillaumeeb** [comment]
Thanks for the review @jhamman, I've taken your comments into account.

**Comment by jhamman** [comment]
Thanks @guillaumeeb!

**Pull request merged**